CONTRIBUTING.md

```diff
@@ -122,15 +122,21 @@ included in the project:
    ```bash
    git pull [--rebase] upstream master
    ```
+6. Check that non-regression tests are still running. Ideally, every new feature should be accompanied by a new test in the `tests` repository:
```

Comment:
documentation doesn't matter, does it ?

Reply:
"
![image](https://github.com/user-attachments/assets/fcf6e1db-bc3b-42c2-ac3d-d6eb1923074a)
, does it ? "